src/components/pages/publicMenu/index.tsx

```diff
@@ -354,7 +389,9 @@ export default function PublicMenuPage({
                                     </p>
                                     <span>{publicMenu.description}</span>
                                 </VStack>
-
+                                {publicMenu.categories.map((_cat) => (
+                                    <div key={_cat.categoryId}>{_cat.title}</div>
+                                ))}
```

Comment:
The underscore is only for used variables

```suggestion
                                {publicMenu.categories.map((category) => (
                                    <div key={category.categoryId}>{category.title}</div>
                                ))}
```